pkg/vex/vex_test.go

```diff
@@ -585,6 +599,37 @@ repositories:
 				}),
 			}),
 		},
+		{
+			name: "check one parent from multiple dependency paths",
+			args: args{
+				// - oci:debian?tag=12
+				//     - pkg:deb/debian/base-files@5.3
+				//        - pkg:deb/debian/bash@5.3
+				//     - pkg:deb/debian/base-files2@5.3
+				//        - pkg:deb/debian/bash@5.3
```

Comment:
This test case looks like multiple parents (bash-files@5.3 and bash-files2@5.3) from a single dependency (bash@5.3). Am I missing something?

Reply:
Right.
But oci:debian?tag=12 is the parent for both bash-files@5.3 and bash-files2@5.3.

For this test case there are two dependency paths:
1.	pkg:deb/debian/bash@5.3 -> pkg:deb/debian/base-files@5.3 -> oci:debian?tag=12
2.	pkg:deb/debian/bash@5.3 -> pkg:deb/debian/base-files2@5.3 -> oci:debian?tag=12

After the first path, we mark oci:debian?tag=12 as visited.
When we process pkg:deb/debian/base-files2@5.3, we see that oci:debian?tag=12 is already visited and return true.